### PR TITLE
Subscribe

### PR DESCRIPTION
**Pull request by manggalang**
hello, im ready for review code

### CODE REVIEW

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.subscribe{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), image-url("bg-bg.jpeg");
+	width: 100%;
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-position: center;
+	margin-top: 92px;
+
+	&__wrap-item{
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: this is useless

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		height: 100%;
+	}
+
+	&__item{
+		display: block;
+		text-align: center;
+		width: 875px;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: this is useless

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		margin-top: 40px;
+	}
+
+	&__button-link{
+		border: 2.5px solid #fff;
+		text-decoration: none;
+	    color: #fff;
+	    font-size: 30px;
+	    text-transform: uppercase;
+	    line-height: 1;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: this is useless, try make it with padding

File: app/views/client/home/_subscribe.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<div class="subscribe">
+	<div class="subscribe__wrap-item">
```
Comment (Fajaru1503) [inline app/views/client/home/_subscribe.html.erb]: use class container-small

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
+.subscribe{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), image-url("bg-bg.jpeg");
+	width: 100%;
```
[the remark this answers is not in the document]
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: okay, allready to fiix that

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	background-size: cover;
+	background-repeat: no-repeat;
```
[the remark this answers is not in the document]
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: okay, im already to deleted this useless code

File: app/assets/stylesheets/pages/home/_subscribe.scss (hunk 1 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: okay, im done to deleted this code

File: app/assets/stylesheets/pages/home/_subscribe.scss (hunk 3 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: already  deleted this code, and i reduce padding top

File: app/views/client/home/_subscribe.html.erb (hunk 4 above)
Comment (manggalang) [reply app/views/client/home/_subscribe.html.erb]: already to adding this code

**Pull request merged**